elyra/pipeline/tests/pipeline_complex.json

```diff
@@ -0,0 +1,555 @@
+{
```

Comment:
I would mix the order of these up a bit.  They look like they're already relatively ordered.

Also, can we trim out the `ui_data` stanzas?  (This could be done separately since I believe there are others the same way)